src/main/webapp/view.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const PARAM_ID = 'id';
+
+/* exported LECTURE_ID */
+window.LECTURE_ID = getLectureId();
+
+initialize();
+
+/**
+ * Initializes the various parts of the lecture viewer.
```

Comment:
Can you elaborate on what the various parts are? Right now it only initializes the discussion section so you could write the comment as:

/** 
 * Initialized the discussion section of the lecture viewer.
 */
// TODO: Initialize the video and transcript.

Reply:
Done